cmake/SDFUtils.cmake

```diff
@@ -52,8 +52,8 @@ endmacro (BUILD_WARNING)
 #################################################
 macro (sdf_add_library _name)
   set(LIBS_DESTINATION ${PROJECT_BINARY_DIR}/src)
-  set_source_files_properties(${ARGN} PROPERTIES COMPILE_DEFINITIONS "BUILDING_DLL")
-  add_library(${_name} SHARED ${ARGN})
+  add_library(${_name} ${ARGN})
```

Comment:
No criticism, just a note for other reviewers that `STATIC` or `SHARED` are not needed explicitly because it is controlled via `BUILD_SHARED_LIBS`.